maat-court-data-api/src/test/java/gov/uk/courtdata/reporder/controller/CCOutcomeControllerTest.java

```diff
@@ -67,33 +67,33 @@ void givenAValidParameters_whenCreateIsInvoked_thenCreateIsSuccess() throws Exce
     @Test
     void givenIncorrectParameters_whenUpdateIsInvoked_thenErrorIsThrown() throws Exception {
         when(validator.validate(any(RepOrderCCOutcome.class))).thenThrow(new ValidationException());
-        mvc.perform(MockMvcRequestBuilders.put(endpointUrl).content("{}").contentType(MediaType.APPLICATION_JSON))
+        mvc.perform(MockMvcRequestBuilders.put(ENDPOINT_URL).content("{}").contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().is4xxClientError());
     }
 
     @Test
     void givenACorrectParameters_whenUpdateIsInvoked_thenShouldSuccess() throws Exception {
         when(validator.validate(any(RepOrderCCOutcome.class))).thenReturn(Optional.empty());
         when(service.update(any())).thenReturn(TestModelDataBuilder.getRepOrderCCOutcomeDTO(1));
-        mvc.perform(MockMvcRequestBuilders.put(endpointUrl).content(objectMapper.writeValueAsString(TestModelDataBuilder.getRepOrderCCOutcome()))
+        mvc.perform(MockMvcRequestBuilders.put(ENDPOINT_URL).content(objectMapper.writeValueAsString(TestModelDataBuilder.getRepOrderCCOutcome()))
                         .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
                 .andExpect(jsonPath("$.id").value("1"));
     }
 
     @Test
-    public void givenIncorrectParameters_whenFindByRepIdIsInvoked_thenErrorIsThrown() throws Exception {
+    void givenIncorrectParameters_whenFindByRepIdIsInvoked_thenErrorIsThrown() throws Exception {
         when(validator.validate(anyInt())).thenThrow(new ValidationException());
-        mvc.perform(MockMvcRequestBuilders.get(endpointUrl + "/reporder/" + INVALID_REP_ID)).andExpect(status().is4xxClientError());
+        mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + "/reporder/" + INVALID_REP_ID)).andExpect(status().is4xxClientError());
```

Comment:
As above, consider using the MockMVC URL variable substitution instead of manually concatenating strings.